### PR TITLE
V8.0.0 Release

### PR DESCRIPTION
**Pull request by SpicyGarlicAlbacoreRoll**

### CODE REVIEW

**Review by williamh890 — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -26,6 +26,12 @@ and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 -->
 ------
+## [v7.2.0](https://github.com/asfadmin/Discovery-asf_search/compare/v7.1.0...v7.2.0)
```
Comment (Cameronsplaze) [inline CHANGELOG.md:29]: Should this be 8.0.0?

File: asf_search/ASFProduct.py [hunk 2]
```diff
@@ -232,6 +238,9 @@ def remotezip(self, session: ASFSession) -> 'RemoteZip':
 
         return remotezip(self.properties['url'], session=session)
 
+    def _has_multiple_files(self):
```
Comment (Cameronsplaze) [inline asf_search/ASFProduct.py:241]: Maybe `_has_additional_files` instead, so it lines up with the `additionalUrls` key it's checking? And it also implies they're "extra, maybe un-required" files more imo

File: tests/yml_tests/test_ASFSearchResults.yml [hunk 3]
```diff
@@ -29,17 +29,17 @@ tests:
...
 
-- Test ASFSearchResults-format L1:
-    results: Fairbanks_L1.yml
+# - Test ASFSearchResults-format L1:
+#     results: Fairbanks_L1.yml
 
-- Test ASFSearchResults-format ERS Stack:
-    results: Fairbanks_ers_stack.yml
+# - Test ASFSearchResults-format ERS Stack:
+#     results: Fairbanks_ers_stack.yml
```
Comment (Cameronsplaze) [inline tests/yml_tests/test_ASFSearchResults.yml:45]: There's a few tests still commented out here and above. Is that supposed to be the case?

Comment (SpicyGarlicAlbacoreRoll) [reply tests/yml_tests/test_ASFSearchResults.yml]: Good catch, went ahead and removed these tests

File: CHANGELOG.md (hunk 1 above)
Comment (SpicyGarlicAlbacoreRoll) [reply CHANGELOG.md]: Updated with the ruff linter PR

File: asf_search/ASFProduct.py (hunk 2 above)
Comment (SpicyGarlicAlbacoreRoll) [reply asf_search/ASFProduct.py]: There was actually a regression by adding this method, instances where downloading ADDITIONAL_FILES would warn and ignore a user provided `fileName` even if `additionalUrls` had only 1 item. Removed it in favor of the original if-statement block.

**Comment by SpicyGarlicAlbacoreRoll** [comment]
(merged with single test casing failing due to timeout, passes on subsequent runs of test suite and running individual test locally)

**Pull request merged**